frontend/src/services/graphql/Feed/listFeedMessages.js

```diff
@@ -13,24 +13,6 @@ export const listFeedMessages = ({ targetUri, targetType, filter }) => ({
       $filter: FeedMessageFilter
     ) {
       getFeed(targetUri: $targetUri, targetType: $targetType) {
-        target {
```

Comment:
I believe we will have the same issue for `listGlossaryAssociations`

Reply:
I would imagine so when one of dashboard or dataset are disabled - is this something that will be resolved as part of PR #731? Otherwise I can open a separate PR once I validate that error as well

Reply:
@noah-paige I was not going to include it in the PR as it is already quite packed. Can we wait for #731 to be merged and then fix the feed?

Reply:
sounds good - let's do that and I will merge this PR